static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js

```diff
@@ -97,6 +99,16 @@ define([
         }
     };
 
+    policies.outbrain = function () {
```

Comment:
Policies would normally describe particular states of the page, rather than individual features. For example, we would have a userLoggedIn policy or a sensitivePage policy (is Outbrain supposed to be disabled on all sensitive content, incidentally?). This allows us to describe what the policies in these cases are. Otherwise I don't think we really take advantage of centralizing this logic.